Make deploy-iqe-cji script compatible with the decoupling of the comp…

ernesgonzalez33 opened the pull request:
…onent name

PR to fix this: https://github.com/gbenhaim/tekton-insights/pull/25#discussion_r1568364824

Victoremepunto commented:
@ernesgonzalez33  BATS tests are failing, because there's a `Dockerfile` in the root, the tests were designed without expecting a Dockerfile in the root - I think when it was added, that broke the tests - you need to :
- Move the Dockerfile to a different location, or rename it
- Adapt the tests' setup and teardown to move the Dockerfile out and restore it after the test runs

ernesgonzalez33 commented:
@Victoremepunto can you check? I added your commits from your branch here

gbenhaim commented:
lgtm

ernesgonzalez33 commented:
@gbenhaim PTAL again. Added a new commit because the component name was being set as an empty commit when the BONFIRE_COMPONENT_NAME was empty, as can be seen in the last check here: https://github.com/gbenhaim/tekton-insights/pull/25/checks. So I changed the code here and it should work now.
